### PR TITLE
Extending the CanImportRecords trait by delimiter guessing

### PR DESCRIPTION
**Pull request by OccTherapist**
Most of the time, the delimiter in CSV files can be guessed by the ammount of appearance in the file.

- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [ ] Visual changes are explained in the PR description using a screenshot/recording of before and after.

### CODE REVIEW

**Comment by danharrin** [comment]
Please change this so that the reader is optionally passed to `getCsvDelimiter()`, which also handles the guessing. Instead of checking twice each time.

File: packages/actions/src/Concerns/CanImportRecords.php [hunk 1]
```diff
@@ -341,7 +358,7 @@ public function getUploadedFileStream(TemporaryUploadedFile $file)
         ]));
     }
 
-    public static function getDefaultName(): ?string
+    public static function getfaultName(): ?string
```
Comment (danharrin) [inline packages/actions/src/Concerns/CanImportRecords.php]: Also error here

**Review by danharrin — CHANGES_REQUESTED** [review]

**Comment by OccTherapist** [comment]
I edited my commit.

**Comment by OccTherapist** [comment]
Thank you very much for the good refactoring.

**Pull request merged**